code/online_challenges/src/codechef/Greedy Puppy/Greedy_Pupy.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
```

Comment:
Everything in file/folder name should be all lowercase separated by underscores. Also the folder name should be the problem code, which is GDOG.